apps/fabric-website/src/pages/Controls/ListCellsPage/docs/ios/ListCellsOverview.md

```diff
@@ -1,7 +1,7 @@
 When displaying fundamental content types like files or people, it's important to provide a consistent appearance across every screen. These list cells have been designed to represent commonly used entities using a consistent system for representing their icons, metadata and secondary states.
 
 <!-- prettier-ignore-start -->
-| Single, Double, and Triple Line Cells | Other Cell Variants |
-| --- | --- |
-| <img src="https://static2.sharepointonline.com/files/fabric/fabric-website/images/controls/ios/listcells/demo-table-view-cells.png" alt="Examples of Various List Cells" style="width: 60%;" /> | <img src="https://static2.sharepointonline.com/files/fabric/fabric-website/images/controls/ios/listcells/other-cells.png" alt="Examples of Various List Cells" style="width: 100%;" /> |
+| Other Cell Variants | Cells with metadata and Grouped style | Single, Double, and Triple Line Cells |
```

Comment:
This should either be all title case or all sentence case. Last I remember from looking at the official style guide, we're actually supposed to use sentence case in headings, but since title case is so widely used in this section of the site, it's probably fine to stick with that for now.
```suggestion
| Other Cell Variants | Cells with Metadata and Grouped Style | Single, Double, and Triple Line Cells |
```

Reply:
You're right, sentence case is the way to go. I'll try to keep that in made and make incremental fixes for that as I seem them.